tests/py_test.py

```diff
@@ -1848,7 +1848,7 @@ def test_nondefault_monster_testnestedflatbuffer(self):
         b.PrependByte(4)
         b.PrependByte(2)
         b.PrependByte(0)
-        sub_buf = b.EndVector(3)
```

Comment:
Can you check the docs for any uses of EndVector too?

Reply:
Oh yeah, I meant to do that. It's definitely used in at least the tutorial.

Reply:
Done.